test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
```

Comment:
can you add a test for if a post has default_includes, but the object you're serializing does not?

I don't think we want default_includes to be 'respected' the whole way down the render chain